documentation/tutorials/1-4-enable-tls.md

```diff
@@ -0,0 +1,79 @@
+## Transport Layer Security (TLS)
+
+[TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) is used to encrypt data exchanged between two applications; it secures data transmitted over the network. Typically, enabling TLS within a highly available database, and between a highly available database and client/server applications, requires domain-specific knowledge and a high level of expertise. Fortunately, the domain-specific knowledge has been encoded into Charmed OpenSearch. This means enabling TLS on Charmed Opensearch is easily available and requires minimal effort on your end.
```

Comment:
I think we need better explanation why TLS is absolutely mandatory for OpenSearch charm. Please ask @Mehdi-Bendriss

Reply:
I've added some lines in explaining why this is necessary, as well as pointing to this doc from the "connecting to opensearch" section.